config/locales/en_LK.yml

```diff
@@ -52,28 +52,50 @@ en-LK:
   bs_below_200_copy:
     numerator: "Patients with RBS/PPBS <200, FBS<126, or Hba1c <7.0 at their last visit in the last 3 months"
     numerator_fbs: "Patients with FBS <126mg/dL, or HbA1c <7% at their last visit in the last 3 months"
+    numerator_dm: "Patients with RBS/PPBS <200, FBS<126, or HbA1c <7.0 at their last visit in the last 3 months"
+    numerator_dm_fbs: "Patients with FBS <126mg/dL or HbA1c <7% at their last visit in the last 3 months"
     reports_card_title: "Blood Sugar <200"
     reports_card_title_fbs: "Blood sugar controlled"
+    reports_card_title_dm: "Blood Sugar <200"
+    reports_card_title_dm_fbs: "Blood sugar controlled"
+    report_card_lower_bar: "blood sugar <200"
+    report_card_lower_bar_fbs:  "FBS <126mg/dL or HbA1c <7%"
     reports_card_subtitle: "Diabetes patients in %{region_name} with blood sugar <200 at their last visit in the last 3 months"
     reports_card_subtitle_fbs: "Diabetes patients in %{region_name} with FBS <126 mg/dL or HbA1c <7% at their last visit in the last 3 months"
+    reports_card_subtitle_dm: "Diabetes patients at %{region_name} registered >3 months ago with blood sugar <200 at their last visit in the last 3 months"
+    reports_card_subtitle_dm_fbs: "Diabetes patients at %{region_name} registered >3 months ago with FBS <126 mg/dL or HbA1c <7% at their last visit in the last 3 months"
     summary: "patients with a blood sugar <200 from"
     summary_fbs: "patients with blood sugar controlled from"
   bs_over_200_copy:
     reports_card_title: "Blood sugar 200-299 or ≥300"
+    reports_card_title_dm: "Blood sugar ≥200"
     reports_card_title_fbs: "Blood sugar uncontrolled"
+    reports_card_title_dm_fbs: "Blood sugar uncontrolled"
```

Comment:
added `reports_card_title_dm_fbs` because the translation provided for `reports_card_title_fbs` in en.yml is different from what expected as per the design so have to make a new key.